### PR TITLE
docs: example code snippet of open in gitpod button & badge

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
## Description
Added Example Code Snippets for Open In Gitpod Button & Badge

## Related Issue(s)

Fixes [Internal Discussion](https://gitpod.slack.com/archives/C01KPEPNBD0/p1660642507592459)

## How to test
- Open Netfliy Preview Link.
- Go to `/docs/getting-started#contribute-with-gitpod-badge`

## Release Notes

```release-note
NONE
```

## Documentation

No

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2584"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2584"></a>

### CODE REVIEW

File: gitpod/docs/getting-started.md [hunk 1]
```diff
@@ -44,24 +44,28 @@ To make it easy for anyone to start a Gitpod workspace based on your project, we
 <img src="https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod" alt="Contribute with Gitpod"/>
 
 ```html
-<a href="https://gitpod.io/#<your-project-url>">
```
Comment (akosyakov) [inline gitpod/docs/getting-started.md:47]: I think your project url is fine. It is just we need to clarify with an example what it means. No need to change template links

Comment (Siddhant-K-code) [reply gitpod/docs/getting-started.md:47]: double code snippets (one for template & another one for example) would look weird. No?

Comment (akosyakov) [reply gitpod/docs/getting-started.md:47]: I did not mean double snippet, a callout to explains what is your-project-url means with an example. 

BTW I liked your-repository-url better 👍🏻

Comment (Siddhant-K-code) [reply gitpod/docs/getting-started.md:47]: Resolved in [`de862b8`](https://github.com/gitpod-io/website/pull/2584/commits/de862b8cde90e4037fdf6457c7f3c577b0e41658)

**Review by akosyakov — APPROVED** [review]

**Pull request merged**